Hai Reveal: pirate weapon plot holes, A09 text walls, Teeneep continuity, and more player engagement (#7658)
endless-sky/endless-sky: data/hai/hai reveal 1 intro.txt

			`	You hug them both in turn and return the greeting. "I thought it seemed odd when they said I was meeting you for this, Katya, but it makes much more sense if you were just easier to get a hold of than Alondo here."`
			`	"Oh, it was always going to be Alondo," she says, "but there's never a bad excuse for traveling with an old friend, especially when you're meeting other friends too."`
			`	"Do we have time for a drink before we leave?" asks Alondo.`
			`	You spend few hours in a pub chatting with Katya and Alondo, and introducing them Yang. After a while, you all begin to leave, and Katya motions you towards her for a private conversation. You follow her to a private booth, and she looks very serious.`
			choice
				`	"What is it Katya?"`
				`	"It's been a while since I've seen you so concerned."`
			`	"Shhh," she says. "There's a problem I need you to solve... discretely. The Free Worlds have seen pirates with Hai weapons for quite a while. Most weapons were on ships, but recently we've found a large stock in a warehouse on an anarchist world.`

> 			`	"I hope ordinary smugglers are at fault, but it may be something more sinister. The Hai government may be arming the pirates, hopefully just as tribute payments. In the south, we know how useful bounties and tributes are for dealing with pirates. From my time as a senator, I can tell you the Republic will not be so understanding.`

Reviewer: ```suggestion
			`	"I hope ordinary smugglers are at fault, but it may be something more sinister. The Hai government may be arming the pirates, hopefully just as tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a senator, I can tell you the Republic will not be so understanding.`
```

Another pure style one, to stress the distinctions in reaction a bit more.

> 			`	"You must find out if Hai government is arming the pirates, and if they are, put a stop to it. Don't tell them the Free Worlds asked you to do it, and whatever you do, don't let the Republic know. It could lead to an explosive diplomatic incident."`

Reviewer: ```suggestion
			`	"You must find out if the Hai government is arming the pirates and, if they are, put a stop to it. Don't tell them the Free Worlds asked you to do it and, whatever you do, don't let the Republic know. It could lead to an explosive diplomatic incident."`
```

Same issue on both of these. Compound predicate with the qualifier dropped in, not a new independent clause. :)

			choice

> 				`	"That's worrying news, Katya. I'll look into it."`

Reviewer 2: ```suggestion
				`	"That's worrying news. I'll look into it."`
```
Katya's name was already used once before. No point using it again.

Reviewer 2: It's like saying "Hey <name> how's the fries?" "It's fine. Thanks." "Welcome, <name>."
